introduction.md

```diff
@@ -5,3 +5,11 @@ Thanks for having me on the dojo.
 I'm a java developer living in Brazil and learning a lot of cool stuff in Code School.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?